services/utils.py

```diff
@@ -21,25 +16,3 @@ def getconn():
 
 
 mypool = pool.QueuePool(getconn, max_overflow=10, pool_size=5, recycle=300)
```

Comment:
Could also resolve the remainder of this file since it's only used in the update service and the corresponding test, or give the 90s their "My X" back.

Reply:
I was waiting for somebody to suggest that! Did it in 2ef813a587cbf4d23fb6f307f921ba4ac99ba384